kubemarine/core/utils.py

```diff
@@ -342,7 +342,7 @@ def rotation_file(self):
                 for i, file in enumerate(files):
                     if i >= not_pack_file and i < delete_old:
                         if 'tar.gz' not in file:
-                            control_plane.sudo(f'tar -czvf {self.dir_path + file + ".tar.gz"} {self.dir_path + file} &&'
+                            control_plane.get_final_nodes().sudo(f'tar -czvf {self.dir_path + file + ".tar.gz"} {self.dir_path + file} &&'
```

Comment:
Here control_plane is a single node which is surely not a node to be removed. So the change is unnecessary.

Reply:
Fixed